packages/create-sitecore-jss/src/templates/nextjs-personalize/src/components/CdpIntegrationScript.tsx

```diff
@@ -1,15 +1,14 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
-import { RouteData } from '@sitecore-jss/sitecore-jss-nextjs';
+import { RouteData, useSitecoreContext } from '@sitecore-jss/sitecore-jss-nextjs';
 import Script from 'next/script';
 import { useEffect } from 'react';
 
 declare const _boxeverq: any;
```

Comment:
`any` types should be removed across the file, it's one of the goals for this task